lib/server/index.js

```diff
@@ -491,6 +495,46 @@ class Relay {
     })
   }
 
+  /**
+   * @param {string} id
+   * @param {string} directory
+   * @param {object} [options]
+   * @param {number} [options.limit]
+   * @param {number} [options.offset]
+   *
+   * @returns {lmdb.RangeIterable}
+   */
+  _searchDirectory (id, directory, options = {}) {
+    const range = { start: '/' + id + directory, end: '/' + id + directory.slice(0, directory.length - 1) + '0', ...options }
+
```

Comment:
```js
    const range = {
      start: path.join('/', id, directory),
      end: directory !== '/'
        ? path.join('/', id, directory.slice(0, directory.length - 1) + '0')
        : undefined,
      ...options
    }
```
to allow searching from entity root dir

Reply:
Updated the endpoint to just accept `start`, `end` and `limit` query params.